website/blog/2023-05-01-package-exports-support.md

```diff
@@ -0,0 +1,229 @@
+---
+title: 'Package Exports Support in React Native'
+authors: [huntie]
+tags: [announcement, metro]
+date: 2023-05-01
+---
+
+# Package Exports Support in React Native
+
+With the release of React Native 0.72, Metro — our JavaScript build tool — now includes experimental support for the `package.json` [`"exports"` field](https://nodejs.org/docs/latest-v18.x/api/packages.html#exports). [When enabled](#enabling-package-exports-experimental), it adds the following functionality:
+
+- [React Native projects will work with more npm packages out-of-the-box](#for-app-developers)
+- [New capabilities for packages to define their API and target React Native](#for-package-maintainers-preview)
+- [Some breaking changes to package resolution (in edge cases)](#breaking-changes)
```

Comment:
```suggestion
- [React Native projects will work with more npm packages out-of-the-box](/blog/2023/05/01/package-exports-support#for-app-developers)
- [New capabilities for packages to define their API and target React Native](/blog/2023/05/01/package-exports-support#for-package-maintainers-preview)
- [Some breaking changes to package resolution (in edge cases)](/blog/2023/05/01/package-exports-support#breaking-changes)
```

Oh, and those post links need to included full URL, so they lead to the post section when clicked from the list view.

Example from earlier post:

https://github.com/facebook/react-native-website/blob/aba3a883819a425cf2e6ced0d49557894e842232/website/blog/2023-01-03-typescript-first.md?plain=1#L12-L14